backend/entities/Property.cs

```diff
@@ -39,6 +39,11 @@ public abstract class Property : BaseEntity
         /// </summary>
         public PropertyClassification Classification { get; set; }
 
+        /// <summary>
+        /// get/set - The encumbrance reason for this property.
+        /// </summary>
+        public string EncumbranceReason { get; set; }
```

Comment:
You should update the `PropertyConfiguration` to set the max length of this field.